src/pages/integrations/gift-cards/confirm-card-purchase/confirm-card-purchase.ts

```diff
@@ -298,8 +300,11 @@ export class ConfirmCardPurchasePage extends ConfirmPage {
 
   private async createTx(wallet, invoice, message: string) {
     const COIN = wallet.coin.toUpperCase();
-    const payProUrl =
-      invoice && invoice.paymentCodes ? invoice.paymentCodes[COIN].BIP73 : null;
+    const paymentCode =
+      COIN == 'eth'
```

Comment:
Same as above. For btc or bch, we should use `invoice.paymentCodes[COIN].BIP73`

Reply:
fixed